### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by til2to**
In this milestone, I have
- Used the HTML&CSS linters
- Created portfolio toolbar section
- Created portfolio headline section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        </ul>
+      </nav>
+    </header>  
+  
+    <main>
+      <section id="headline">
+        <article class="headlines">
+          <h1> Hey There. <br> I'm Dennis </h1>
+          <h2> I'm a Software Developer </h2>
+        </article>
```
Comment (christianbiring1) [inline index.html]: - [ ] `article` doesn't have any semantic meaning here. Kindly remove that.
`You can check` [here](https://www.w3schools.com/tags/tag_article.asp) `when to use the article tag`
`hint: You may use generic div inf you want to`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  
+        <article class="brand">
+          <p>
+            I can help you build a product, 
+            feature or website Look through some of my 
+            work and experience! If you like what you 
+            see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+        </article>
```
Comment (christianbiring1) [inline index.html]: - [ ] `article` doesn't have any semantic meaning here. Kindly remove that.
`You can check` [here](https://www.w3schools.com/tags/tag_article.asp) `when to use the article tag`
`hint: You may use generic div inf you want to`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          <li><img src="./images/mail.png" class="mail" alt="image"><span>></span></li>
+        </ul>
+      </nav>
+    </header>  
+  
+    <main>
+      <section id="headline">
+        <article class="headlines">
+          <h1> Hey There. <br> I'm Dennis </h1>
+          <h2> I'm a Software Developer </h2>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly use the header `h1` to grab all the introductory sections for semantic purposes.

`Hint: <h1>Hey There. I'm Dennis I'm a Software Developer</h1>`

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @til2to , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose



## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good UI

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Deleting all the sections that are not applicable to this project in your README file as the `Deployment`
- Also you may consider providing a thorough walkthrough on the `Get started`
`HINT:` `How to get the local copy of the repos, how to navigate in the repos locally, how to run it locally`. `You will need to provide the commands though`


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+        <section class="social-media">
+          <ul>
+            <li><a href="#"><img src="./images/one.png" alt=""></a></li>
+            <li><a href="#"><img src="./images/Frame.png" alt=""></a></li>
+            <li><a href="#"><img src="./images/three.png" alt=""></a></li>
+            <li><a href="#"><img src="./images/four.png" alt=""></a></li>
+            <li><a href="#"><img src="./images/five.png" alt=""></a></li>
+          </ul>
+        </section>
```
Comment (leonardodiasb) [inline index.html]: - [ ] As the previous code reviewer mentioned you should not use the `article` tag to wrap and position elements inside a `section` tag, but you should not use the `section` tag to wrap and position elements too. The `div` tag is more appropriate for that function while the `section` tag should group everything that belongs to that section. Kindly change the nested `section` tags to `div` tags. Please take a look [at this article](https://stackoverflow.com/questions/7041283/nesting-html5-section-tags) for more info.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @til2to, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - All linters checks are green ✔️
 - Descriptive PR title and summary ✔️ 
 - Good UI ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (til2to) [reply index.html]: Alright

**Review by ichala — APPROVED** [review]
Hi @til2to ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

-Requirements Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**